### DISCUSSION TITLE
Welcome to Solara Discussions!

### DISCUSSION THREAD

**Discussion by quivings** [quivings/Solara#157] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by buhhhhg** [comment]
buh

**Comment by nukezx** [comment]
When will the solara update come?

**Reply by Sigma146** [reply]
stfu let qui cook lil bro

**Comment by depthso** [comment]
If you add setreadonly and getrawmetatable I will kiss your feet or shoes. Polish your shoes until they are shiny new. Make breakfast for you. Make dinner for you. Dk the house chores for you

**Reply by itzzexcel** [reply]
Absolutely.

**Comment by Minime11223** [comment]
im not sure what im doing wrong? is it for the 'store' version bc when i try to inject it tells me to 'open roblox' allso if i try it on the other version it just crashes? 
thanks 
Edit: why cant i type in the general on discord XD

**Reply by ImFloriz** [reply]
[0] "store version": you need to be on the web version, and need to be on the LIVE channel
[1] "why cant i type": you need to boost the solara server
